bugbug/models/duplicate.py

```diff
@@ -48,7 +48,12 @@ def __init__(self, training_size=14000, lemmatization=False):
 
         self.extraction_pipeline = Pipeline(
             [
-                ("bug_extractor", bug_features.BugExtractor([], cleanup_functions)),
+                (
+                    "bug_extractor",
+                    bug_features.BugExtractor(
+                        [bug_features.is_same_product()], cleanup_functions
```

Comment:
Nit: define the extractors earlier, before cleanup_functions (to have the same style as the other models).